src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,610 @@
+    "last_token_uri_sync": "2023-02-19T19:12:09.471Z",
+    "last_metadata_sync": "2023-02-19T19:12:18.080Z",
+    "minter_address": "0xdbdeb0895f3681b87fb3654b5cf3e05546ba24a9",
+    "possible_spam": false,
+    "uri_meta": {
+      "image": "https://public.nftstatic.com/static/nft/res/4df0a5da04174e1e9be04b22a805f605.png",
+      "token_name": "Nebula Nodes",
+      "description": "Interchain nodes continuously exchange blockchain data amongst each other, supporting blockchain interoperability.",
+      "attributes": null,
+      "animation_url": null
+    }
+  },
+  "id": null
+}
+```
+</CollapsibleSection>
+
+
+## Withdraw NFTs {{label : 'withdraw_nft', tag : 'API-v2'}}
+
+<Note>
+To withdraw NFTs, you must [activate the coin](/atomicdex/api/legacy/coin_activation) which holds the NFTs first.
+</Note>
+
+### Request Parameters
+
+| Parameter     | Type             | Description                                                                                               |
+| ------------- | ---------------- | ----------------------------------------------------------------------------------------------------------|
+| url           | string           | URL link to [Moralis API](https://docs.moralis.io/) (or proxy).                                           |
+| withdraw_type | object           | A standard [WithdrawNftType](/atomicdex/api/v20#withdraw-nft-type) object.                                |
+
+<Note>
+When the `type` value in `WithdrawType` is `withdraw_erc721` it means the NFT is absolutely unique - it has only 1 owner and the owner can own only 1 NFT which has this `token_id` in its `token_address` (also referred to as contract address).
+When the `type` value in `WithdrawType` is `withdraw_erc1155`, it means that it is possible for more than 1 user to own one or more of the same NFT (with an identical `token_id`).
+Due to this difference, the `amount` and `max` fields are only used the `type` value in `WithdrawType` is `withdraw_erc1155`.
+</Note>
+
+
+#### 📌 ERC721 Withdraw Example
+
+<CodeGroup title="" tag="POST" label="withdraw_nft">
+```json {{ mm2MethodDecorate : true }}
+{
+  "userpass": "MM2_RPC_PASSWORD",
+  "method": "withdraw_nft",
+  "mmrpc": "2.0",
+  "params": {
+    "url": "https://moralis-proxy.komodo.earth",
+    "withdraw_type": {
+      "type": "withdraw_erc721",
+      "withdraw_data": {
+        "chain": "BSC",
+        "to": "0x6FAD0eC6bb76914b2a2a800686acc22970645820",
+        "token_address": "0xfd913a305d70a60aac4faac70c739563738e1f81",
+        "token_id": "214300044414"
+      }
+    }
+  }
+}
+```
+</CodeGroup>
+
+<DevComment>
+May change in future. See https://github.com/KomodoPlatform/komodo-docs-mdx/pull/42#discussion_r1197745312
+Right now `internal_id` is always zero and actually I dont save TransactionNftDetails in db as we use info from moralis (NftTransferHistory), but may be in the future. I will need it...
```

Comment:
sorry will this text be visible on the website?:D
I would like to have an anonymous text like this:

Right now `internal_id` is always zero and actually we dont save `TransactionNftDetails` in db as we use info from moralis (`NftTransferHistory` structure). However, perhaps in the future, in the process of refactoring, we might stop using the moralis API and start using the API of decentralized platforms. In this case this field may be needed.

Reply:
DevComment tags are not shown to the public, just a way for us to add comments to docs for later.